.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+  "name": "Ruby 2.5 on Debian",
+  "image": "ruby:2.5-slim-buster", // Specify the Ruby 2.5 image
+  "postCreateCommand": "RUBYGEMS_VERSION=3.3.27\ngem install rubygems-update -v ${RUBYGEMS_VERSION} > /dev/null 2>&1\nupdate_rubygems > /dev/null 2>&1\n", // Optional: Run this command after the container is created
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "./apt-install": {}
+  },
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "ruby --version",
```

Comment:
Could you remove needless comments?

Reply:
Yes, but as you are new to devcontainers you should be aware that this is not vanilla JSON.  Devcontainers use a newer spec called "JSON with comments", and known as `JSONC` in linters.

- https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers#editing-the-devcontainerjson-file

Reply:
Oh!